feat: DIA-1716: Google AI Studio (Gemini) support in Prompts

matt-bernstein opened the pull request:
### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)



#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend



### Describe the reason for change
_(link to issue, supportive screenshots etc.)_



#### What does this fix?
_(if this is a bug fix)_



#### What is the new behavior?
_(if this is a breaking or feature change)_



#### What is the current behavior?
_(if this is a breaking or feature change)_



#### What libraries were added/updated?
_(list all with version changes)_



#### Does this change affect performance?
_(if so describe the impacts positive or negative)_



#### Does this change affect security?
_(if so describe the impacts positive or negative)_



#### What alternative approaches were there?
_(briefly list any if applicable)_



#### What feature flags were used to cover this change?
_(briefly list any if applicable)_



### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)



### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit



### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_


- https://github.com/HumanSignal/label-studio-sdk/pull/393 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/12920835621)

Commit:
feat: DIA-1715: VertexAI Gemini model support

Commit:
Merge branch 'develop' of github.com:heartexlabs/label-studio into fb-dia-1715

matt-bernstein committed:
Merge remote-tracking branch 'origin/develop' into fb-dia-1715

Commit:
Merge branch 'develop' of github.com:heartexlabs/label-studio into fb-dia-1715

Commit:
Merge branch 'fb-dia-1715' of github.com:heartexlabs/label-studio into fb-dia-1715

matt-bernstein committed:
feat: DIA-1716: Google AI Studio support in Prompts

matt-bernstein committed:
add gemini to enum

robot-ci-heartex commented:
Follow Merge downstream workflow has been failed.
> [Workflow run](https://github.com/HumanSignal/label-studio-enterprise/actions/runs/12836536703)

robot-ci-heartex commented:
Follow Merge downstream workflow has been failed.
> [Workflow run](https://github.com/HumanSignal/label-studio-enterprise/actions/runs/12930704020)

matt-bernstein committed:
Merge remote-tracking branch 'origin/develop' into fb-dia-1716

matt-bernstein committed:
Merge remote-tracking branch 'origin/develop' into fb-dia-1716

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13013135048

matt-bernstein committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13017298341

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13021748106)
> Successfully merged:  delete mode 100644 web/tsconfig.json

matt-bernstein committed:
Merge branch 'develop' into 'fb-dia-1716'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13021748106

matt-bernstein committed:
Merge remote-tracking branch 'origin/develop' into fb-dia-1716

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13041443119

matt-bernstein committed:
Merge remote-tracking branch 'origin/develop' into fb-dia-1716
